.changeset/violet-steaks-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@solidjs/start": minor
```

Comment:
I was kinda expecting this one to be a **patch** since it's not changing any API nor is it adding a new feature.

wdyt?